<?xml version="1.0"?>

<launch>
    <node pkg="sgba_simulation" type="SGBASimulation" name="SGBASimulation" output="screen">
        <param name="map_file" type="string" value="/maps/map_40_40.txt"/>
        <param name="log_file_path" type="string" value="/path_log/40_40_world/three_uavs"/>
        <param name="uav_num" type="int" value="3"/>
    </node>
</launch>